### ISSUE TITLE
Does cq-prolyfill inlined my style?

### ISSUE THREAD

**Issue by ariona** [ausi/cq-prolyfill#31]
I was wondering why my sourcemaps isn't working, since the style is being inlined in the head but i haven't done that.

Turn out the cq-prolyfill did that, is there any way to still use sourcemap while using cq-prolyfill.

**Comment by ausi** [comment]
Yes, *cq-prolyfill* has to convert the `:container()` parts of the CSS so that the browser can parse it. You can prevent that by using the [PostCSS plugin](https://github.com/ausi/cq-prolyfill/blob/master/docs/postcss.md) or [Sass mixin](https://github.com/ausi/cq-prolyfill/issues/17#issuecomment-183756013). And you should [enable CORS](https://github.com/ausi/cq-prolyfill/blob/master/docs/cors.md) if the stylesheet is loaded from a different domain.
